FabricLogging.py

```diff
@@ -29,6 +29,7 @@
     'NOTSET': NOTSET,
 }

```

Comment:
The idea behind leaving it in the code is based on 2 things:

1. These values are not up for debate, they are hardcoded because they are not supposed to ever change. This ensures that any project that implements this library will always have these values in common, so that developers won't have to adapt to new levels of errors when entering a new project.
2. In order to keep the library simple and easy to use i want to reduce the amount of files as much as possible.

Thoughts on the above are welcome

Reply:
Makes perfect sense. So what do you think about still implementing an option to set it from the config file and still setting those values as default ones, that approach will need a little code to restructure.